packages/nc-gui-v2/pages/[projectType]/view/[viewId].vue

```diff
@@ -9,11 +10,13 @@ import {
   useRoute,
   useSharedView,
 } from '#imports'
+import { extractSdkResponseErrorMsg } from '~/utils'
 
 definePageMeta({
   public: true,
```

Comment:
Can you remove this? I accidently added this parameter but you seem to use another one 😄